blocks/header/header.css

```diff
@@ -359,7 +404,7 @@ header nav .nav-sections .nav-drop {
     padding: 12px 0;
     white-space: unset;
     text-wrap: wrap;
-    width: 30ch;
+    width: 22dvw;
```

Comment:
Looks like [Firefox still doesn't support `dvw`](https://caniuse.com/?search=dvw)

Reply:
@sdmcraft It does :) What you're looking at is dvw() method not the unit. If you scroll down you'll see it is supported and now recommended over vw as it dynamically adjusts to viewport